pkgs/development/compilers/go/1.12.nix

```diff
@@ -96,6 +96,12 @@ stdenv.mkDerivation rec {
     # Disable cgo lookup tests not works, they depend on resolver
     rm src/net/cgo_unix_test.go
 
+    # Disable TestGcSys because it's flakey in our tests, but the failure is not
+    # reproducible by multiple people in other environments.
+    # See https://github.com/NixOS/nixpkgs/issues/68361#issuecomment-537849272 and following
+    # NOTE: Try re-enabling for releases newer than 1.12.9
+    sed -i '/TestGcSys/areturn' src/runtime/gc_test.go
```

Comment:
Can this be done on the affected Arch only? I believe `i686` is the culprit.

Reply:
@kalbasit: I was considering this too initially, but then thought that since we've seen this failure on x86_64 too when run on ofborg might as well just get this out of the way for the release and keep the test in on master and everywhere else to see if this stops happening with some newer version of GO. We saw such a failure when trying the upstream patch: https://github.com/NixOS/nixpkgs/pull/69359/checks?check_run_id=234367395 .

Reply:
I see! Fair enough then.